charts/gardener-extension-provider-azure/templates/rbac.yaml

```
@@ -55,6 +61,16 @@ rules:
   verbs:
   - get
   - update
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  resourceNames:
+  - remedy-controller-azure-leader-election
+  verbs:
+  - get
+  - watch
+  - update
```

```suggestion
  - remedy-controller-azure-leader-election
  verbs:
  - get
  - update
```

We can add the `remedy-controller-azure-leader-election` resource name to the previous rule that already gives `get` &`update` verbs to specific leases, the `watch` verb is given for all leases in https://github.com/gardener/gardener-extension-provider-azure/blob/eef394863a578d18887c4d3cf777d6d32ce00ff6/charts/gardener-extension-provider-azure/templates/rbac.yaml#L40-L47